src/main/java/Tommi.java

```diff
@@ -0,0 +1,96 @@
+import java.util.Scanner;
+
+public class Tommi {
+    private static final int MAX_TASKS = 100;
+    private static final String[] tasks = new String[MAX_TASKS];  // Array to store tasks
+    private static final boolean[] isCompleted = new boolean[MAX_TASKS];  // Array to store task completion status
+    private static int taskCount = 0;  // Counter to keep track of the number of tasks
```

Comment:
Why not use a Class to represent all instances of tasks?

Reply:
I concur